plugins/WindowManager/TopLevelWindowModel.cpp

```diff
@@ -38,7 +39,8 @@ Q_LOGGING_CATEGORY(TOPLEVELWINDOWMODEL, "toplevelwindowmodel", QtInfoMsg)
 namespace unityapi = unity::shell::application;
 
 TopLevelWindowModel::TopLevelWindowModel()
-    : m_nullWindow(createWindow(nullptr)),
+    : m_nullWindow(createNullWindow()),
+      m_maxWindowId(1000000),
```

Comment:
I presume this is changed to a non-const int to be able to initialize or set to a very small value for tests. However, the old comment below is I think wrong and should be removed, and we should use `std::numeric_limits<int>::max()` here, as Qt seems to use a 32-bit int for the QML int.

Reply:
It is to allow for messing with it in tests, yes, and now I see the QML documentation says "The possible int values range from around -2000000000 to around 2000000000", the range of a 32-bit signed integer. I'll make that change.

Reply:
Got it, and tests passed locally. Now we wait for `-j1` tests to do their work...